numpy_tasks.ipynb

```diff
@@ -9,14 +9,34 @@
     "https://docs.google.com/spreadsheets/d/1v1_X6zRlwRVjULZSrRvBYAG_raUPjTI0Ceruwh6vNXw/edit#gid=2019320840\n"
```

Comment:
<p>Avoid <code>append</code> by calling  <code>where</code>  not for the whole array</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='109'/>